src/openxr/OpenXRApi.cpp

```diff
@@ -1534,21 +1505,26 @@ void OpenXRApi::uninitialize() {
 	if (session != XR_NULL_HANDLE) {
 		xrDestroySession(session);
 		session = XR_NULL_HANDLE;
+
+		for (XRExtensionWrapper * wrapper : registered_extension_wrappers) {
+			wrapper->on_session_destroyed();
+		}
 	}
 	if (instance != XR_NULL_HANDLE) {
 		xrDestroyInstance(instance);
 		instance = XR_NULL_HANDLE;
+
+		for (XRExtensionWrapper * wrapper : registered_extension_wrappers) {
```

Comment:
Same here, should we be calling before destroying the instance as we may need the instance to clean up...